README.md

```diff
@@ -131,6 +131,10 @@ On Linux, the following tools should be installed:
 - [`genhtml`](http://linux.die.net/man/1/genhtml): for creating reports in html
     format from `lcov` output, provided by the `lcov` package on Ubuntu.
 
+On Linux or Windows, the following tool should be installed and available on your path:
```

Comment:
Ninja is just one alternative -- you can also use make.  Consider rewording this to mention CMake generators and link to the CMake [documentation](https://cmake.org/cmake/help/v3.0/manual/cmake-generators.7.html).